kubernetes-model/pkg/schemagen/generate.go

```diff
@@ -152,9 +152,13 @@ func (g *schemaGenerator) javaType(t reflect.Type) string {
 	//as the name of Resources are same in both places.
 
 	if t.Kind() == reflect.Struct && ok {
-		if g.qualifiedName(t) == "kubernetes_extensions_RunAsUserStrategyOptions" || strings.HasPrefix(g.qualifiedName(t), "kubernetes_rbac_") {
+		if g.qualifiedName(t) == "kubernetes_extensions_RunAsUserStrategyOptions" {
 			return pkgDesc.JavaPackage + "." + "Kubernetes" + t.Name()
 		}
```

Comment:
This is related to #1152. I've tried to generate kube-schema.json file using schema generator tool from master branch, but it will not include changes introduced in #1152. RBAC Kuberentes resources are still prefixed with "Kubernetes" and Openshift role related resources doesn't have "Openshift" prefix. I'm not why change in schema generator was not part of #1152.

Reply:
oh okay